$OpenBSD: patch-src_desktop_widgets_userlistwidget_h,v 1.1 2018/05/29 19:48:53 rsadowski Exp $

"The order in which files are placed in mocs_compilation.cpp has changed
in CMake 3.11 compared to CMake 3.10. This shows up places where not all
of the includes needed for a bit of code, are included -- because by chance,
everywhere else they are already included by the time the compiler gets to
the offending header. In switching the order around, compiles fail."

-- https://reviews.freebsd.org/D14770

Index: src/desktop/widgets/userlistwidget.h
--- src/desktop/widgets/userlistwidget.h.orig
+++ src/desktop/widgets/userlistwidget.h
@@ -23,6 +23,7 @@
 
 #include <QWidget>
 #include <QItemDelegate>
+#include <QItemSelection>
 
 class Ui_UserBox;
 
